### DISCUSSION TITLE
BCL website is down

### DISCUSSION THREAD

**Discussion by Feriel02** [BuildingComponentLibrary/bcl-manifest#22] [General]
Hello, SOS I am currently working on a project on openstudio and i need some measures to download but the BCL is Down from yesterday any help ????

**Comment by kflemin** [comment]
Hi @Feriel02, thanks for letting us know. The BCL should be back online now.

**Reply by Feriel02** [reply]
yess it is working now thanks for your response

**Comment by roruizf** [comment]
Hello,

It seems that the BCL is down again. I've been trying to download measures from the OpenStudio application since yesterday, but it's not displaying any measures or components. Additionally, the URL https://bcl.nrel.gov/browse is returning an "Internal Server Error."

Could you please help me?

**Reply by Smit199782** [reply]
Same, I have the same issue. Just started using OpenStudio and I thought the software is not useful, but the server is not working so it is showing me errors.

**Reply by kflemin** [reply]
I just checked it right now and the BCL is up and running...are you still seeing this issue?

**Comment by Ski90Moo** [comment]
I am able to download directly from the BCL website.

I can also get measures via the OpenStudio interface.  However, I had to
close and reopen the App before they would appear in the measures list.

On Tue, Apr 29, 2025 at 5:25 AM Fabian22550 ***@***.***>
wrote:

> Hello, it seems the bcl is down again? I cant reach the site
> https://bcl.nrel.gov/ at all.
> Im thankful for any help
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BuildingComponentLibrary/bcl-manifest/discussions/22#discussioncomment-12979850>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQUKBFGD3VROG3HC7VNGKHD235VNXAVCNFSM6AAAAAB4CZ7356VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTEOJXHE4DKMA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID:
> <BuildingComponentLibrary/bcl-manifest/repo-discussions/22/comments/12979850
> @github.com>
>


-- 
Best Regards,
Mike Lovejoy P.E.
Helix Energy Partners LLC
Helix, OR
http://www.helix-engineers.net/
<https://deref-mail.com/mail/client/aJLmMsp2k0Y/dereferrer/?redirectUrl=http%3A%2F%2Fwww.helix-engineers.net%2F>
P:541-379-0271

**Reply by Fabian22550** [reply]
Hi, thanks for the quick reply! the issue of not finding the website was  due to my own error, firewall issues :D Downloading the measures through the OpenStudio Interface works fine for me. But still my problem remains , because i have found a link from the unmethours forum leading me to this measure:

https://bcl.nrel.gov/node/82786

to change the scheduled temperature to be the operative temperature.

I urgently need that function for my project. If you have any idea how i can access these let me know.

**Comment by Ski90Moo** [comment]
What is the name of the measure? Or a link to the unmet hours thread?  The
link you supplied is broken.

On Tue, Apr 29, 2025 at 7:31 AM Fabian22550 ***@***.***>
wrote:

> Hi, thanks for the quick reply! the issue of not finding the website was
> due to my own error, firewall issues :D Downloading the measures through
> the OpenStudio Interface works fine for me. But still my problem remains ,
> because i have found a link from the unmethours forum leading me to this
> measure:
>
> https://bcl.nrel.gov/node/82786
>
> to change the scheduled temperature to be the operative temperature.
>
> I urgently need that function for my project. If you have any idea how i
> can access these let me know.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BuildingComponentLibrary/bcl-manifest/discussions/22#discussioncomment-12981887>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQUKBFEFMNW6F2RFGOS6ZTD236ELVAVCNFSM6AAAAAB4CZ7356VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTEOJYGE4DQNY>
> .
> You are receiving this because you commented.Message ID:
> <BuildingComponentLibrary/bcl-manifest/repo-discussions/22/comments/12981887
> @github.com>
>


-- 
Best Regards,
Mike Lovejoy P.E.
Helix Energy Partners LLC
Helix, OR
http://www.helix-engineers.net/
<https://deref-mail.com/mail/client/aJLmMsp2k0Y/dereferrer/?redirectUrl=http%3A%2F%2Fwww.helix-engineers.net%2F>
P:541-379-0271

**Reply by Fabian22550** [reply]
Its this page.
https://unmethours.com/question/38682/thermostat-on-the-base-of-operative-temperatureurgent/

Im trying to change the thermostat setpoint to be the "Thermostat:OperativeTemperature"

Normally the "Thermostat" Objects sets the air temperature right?

**Comment by Ski90Moo** [comment]
This measure appears to be from the old version of the BCL.  If it was an
NREL measure, it would most likely have been forwarded to the new BCL.  A
cursory search and I cannot find it.  So, it was probably a third party
measure.  You could try to email Katherine to see if she still has it:

https://unmethours.com/question/99175/finding-measure-from-old-bcl/

On Tue, Apr 29, 2025 at 8:15 AM Fabian22550 ***@***.***>
wrote:

> Its this page.
>
> https://unmethours.com/question/38682/thermostat-on-the-base-of-operative-temperatureurgent/
>
> Im trying to change the thermostat setpoint to be the
> "Thermostat:OperativeTemperature"
>
> Normally the "Thermostat" Objects sets the air temperature right?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BuildingComponentLibrary/bcl-manifest/discussions/22#discussioncomment-12982396>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQUKBFGSXNXXHD5H3737ZVD236JQFAVCNFSM6AAAAAB4CZ7356VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTEOJYGIZTSNQ>
> .
> You are receiving this because you commented.Message ID:
> <BuildingComponentLibrary/bcl-manifest/repo-discussions/22/comments/12982396
> @github.com>
>


-- 
Best Regards,
Mike Lovejoy P.E.
Helix Energy Partners LLC
Helix, OR
http://www.helix-engineers.net/
<https://deref-mail.com/mail/client/aJLmMsp2k0Y/dereferrer/?redirectUrl=http%3A%2F%2Fwww.helix-engineers.net%2F>
P:541-379-0271
